src/openrct2/ride/Vehicle.cpp

```diff
@@ -9282,16 +9282,14 @@ static int32_t vehicle_update_track_motion_mini_golf(rct_vehicle* vehicle, int32
 
 loc_6DCE68:
     _vehicleMotionTrackFlags |= VEHICLE_UPDATE_MOTION_TRACK_FLAG_VEHICLE_AT_STATION;
-    regs.al = vehicle->track_x >> 5;
-    regs.ah = vehicle->track_y >> 5;
-    regs.dl = vehicle->track_z >> 3;
```

Comment:
maybe if you did
```c++
auto vehicleTileCoords = TileCoordsXYZ{CoordsXYZ{vehicle->track_x, vehicle->track_y, vehicle->track_z}};
```

It would make the comparisons below more readable